awx/ui_next/src/components/FormField/sortErrorMessages.js

```diff
@@ -52,6 +52,9 @@ function parseFieldErrors(obj, formValues) {
       }
       formErrors = formErrors.concat(parsed.formErrors);
     }
+    if (typeof formValues[key] === 'boolean') {
```

Comment:
I wonder if this should just be an `else` block to catch any and all "other" types that could occur

Reply:
I tried the catch all - else, but the error was not displayed.